install/Knative-with-ICP.md

```diff
@@ -0,0 +1,185 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. See [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html) in the IBM Knowledge Center for install instructions.
```

Comment:
```suggestion
 Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. 
 
 [Complete all the steps in the IBM Cloud Private installation instructions to create your standard? or HA? cluster](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html), for example:
```

Does it matter what type of cluster (standard vs HA)?

Reply:
The [KC topic](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html) is standard cluster installation.